templates/helmfile/monster_comand_center_dagster.yaml

```diff
@@ -0,0 +1,76 @@
+---
+repositories:
+  - name: dagster
+    url: https://dagster-io.github.io/helm
+  - name: datarepo-helm
+    url: https://broadinstitute.github.io/datarepo-helm
+
+# helm releases to be deployed
+releases:
+  # sets up a pod security policy for the given service account
+  # (required by our org's broader security policy)
+  - name: monster-psp    # release name
+    namespace: dagster   # target namespace
+    chart: datarepo-helm/serviceaccount-psp   # chart name
+    missingFileHandler: Warn
+    values:
+      - serviceAccount:
+          name: monster-dagster
+  - name: monster-dagster-secrets
+    namespace: dagster
+    chart: datarepo-helm/create-secret-manager-secret
+    missingFileHandler: Warn
+    values:
+      - secrets:
+          - secretName: monster-dagster-secrets
+            nameSpace: dagster
+            vals:
+              # any secrets need to be registered here
+              #########
+              # DAP
+              #########
+              - kubeSecretKey: REDCAP_ENV_API_TOKEN
```

Comment:
moving forward, all unique secrets for each project would get added here?

Reply:
Yeah; ideally we'd segment these into different namespaces rather than dumping them all into `dagster`, but this is fine for now IMO